build.gradle.kts

```diff
@@ -27,6 +27,7 @@ plugins {
     alias(libs.plugins.errorprone) apply false
     alias(libs.plugins.gradle.versions) apply false
     alias(libs.plugins.spring.dependency.management) apply false
+    id("sentry.javadoc.aggregate")
```

Comment:
It's probably best to use the full `io.sentry.**` namespace instead of `sentry.**` for the plugin ids.

Reply:
It’s a good point, i wanted to distinguish this internal plugin from externally published plugins. I’ll do the switch but maybe its worth discussing how to distinguish internal vs external plugins.